java.lang.IllegalArgumentException: port out of range:-9000
	at java.net.InetSocketAddress.checkPort(InetSocketAddress.java:123)
	at java.net.InetSocketAddress.<init>(InetSocketAddress.java:204)
	at play.core.server.NettyServer$$anonfun$10.apply(NettyServer.scala:134)
	at play.core.server.NettyServer$$anonfun$10.apply(NettyServer.scala:131)
	at scala.Option.map(Option.scala:145)
	at play.core.server.NettyServer.<init>(NettyServer.scala:131)
	at play.core.server.NettyServer$$anonfun$mainDev$1.apply(NettyServer.scala:310)
	at play.core.server.NettyServer$$anonfun$mainDev$1.apply(NettyServer.scala:308)
	at play.utils.Threads$.withContextClassLoader(Threads.scala:18)
	at play.core.server.NettyServer$.mainDev(NettyServer.scala:307)
	at play.core.server.NettyServer$.mainDevHttpMode(NettyServer.scala:303)
	at play.core.server.NettyServer.mainDevHttpMode(NettyServer.scala)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$1.apply(PlayRun.scala:233)
	at play.PlayRun$$anonfun$playRunTask$1$$anonfun$apply$1.apply(PlayRun.scala:91)
	at scala.Function7$$anonfun$tupled$1.apply(Function7.scala:35)
	at scala.Function7$$anonfun$tupled$1.apply(Function7.scala:34)
	at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47)
	at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:42)
	at sbt.std.Transform$$anon$4.work(System.scala:64)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:237)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:237)
	at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:18)
	at sbt.Execute.work(Execute.scala:244)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:237)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:237)
	at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:160)
	at sbt.CompletionService$$anon$2.call(CompletionService.scala:30)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:695)
[0m[[31merror[0m] [0m(compile:[31mrun[0m) java.lang.reflect.InvocationTargetException[0m
